src/pages/AuthentificationPage/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react"
+import "./Authentification.scss"
+
+import SignInUp from "../../components/SignInUp"
```

Comment:
fa o mapa **components** in interior la **src/pages/AuthentificationPage** si muta acolo componenta **SignUp** asta e ceva ce nu va fi utilizat in alta parte, deci tre sa fie o componenta local pentru pagina data